lib/player/player.js

```diff
@@ -736,6 +741,36 @@ shaka.player.Player.prototype.setPlaybackRate = function(rate) {
 
 
 /**
+ * @param {Object} restrictions An Object containing the restrictions
+ *     on video tracks that you wish to set. For example, passing
+ *     {'maxBandwidth': 700000, 'minBandwidth': 200000} will restrict
+ *     playback to video tracks with bandwidth between 700000 and 200000.
+ * @export
+ */
+shaka.player.Player.prototype.setRestrictions = function(restrictions) {
+  for (var key in restrictions) {
```

Comment:
Okay, on rereading, I think you are referring to this? 

var r = player.getRestrictions();
r.maxBandwidth = 5;
player.setRestrictions(r);

So set restrictions would instead of iterating through the keys, getRestrictions() which will return a copy of the restrictions object. Then the properties can be set on this copy and the old restrictions can be replaced with this new one. 

I like this, my one hang up is explicitly setting properties like : 

r.maxBandwidth = 5;

The advantage I would think of iterating is that if new restrictions are supported in the future we won't have to change this method.

What are your thoughts?